client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -84,7 +85,8 @@ export class DraggablePanelComponent implements OnInit, OnChanges, OnDestroy {
     private router: Router,
     private agolService: AGOLService,
     private commonUtilityService: CommonUtilityService,
-    private appConfigService: AppConfigService
+    private appConfigService: AppConfigService,
+    public componentFactoryResolver: ComponentFactoryResolver,
```

Comment:
This is used by identify-panel. which extends DraggablePanelComponent

Reply:
We could, but I think it means we will create a constructor in identify-panel with 
    super(publishedIncidentService, cdr, http, mapConfigService, router, agolService, commonUtilityService, appConfigService)... etc 
 but In fact we only need to use the componentFactoryResolver from what we extended from draggable panel, I think this could be more cleaner?